web/docs/setup/tune.md

```diff
@@ -230,45 +173,75 @@ vast:
         partition-index: false
 ```
 
-## Shutdown
+### Select the store format
 
-The `stop` command gracefully brings down a VAST server that has been started
-with the `start` command.
+VAST arranges data in horizontal partitions for sharding. Each partition has a
+unique schema and is effectively a concatenation of batches. The *store* inside
+a partition holds this data and can have multiple formats:
 
-It is also possible to send a signal `SIGINT(2)` to the `vast` process instead
-of using `vast stop`, but in only works on the same machine that runs the
-server process. We recommend using `vast stop`, as it also works over the wire.
+1. **Feather**: writes Apache Feather V2 files
+2. **Parquet**: writes [Apache Parquet](https://parquet.apache.org/) files
 
-The `stop` command blocks until the server process has terminated, and returns
-a zero exit code upon success, making it suitable for use in launch system
-scripts.
+VAST defaults to the `feather` store. To use the Parquet store, adjust
+`vast.store-backend`:
 
-:::note
-The server waits for ongoing import processes to terminate before shutting down
-itself. In case an import process is hanging, you can always terminate the
-hanging process manually to shutdown the server.
+```yaml
+vast:
+  plugins:
+    - parquet
+  store-backend: parquet
+```
+
+There's an inherent space-time tradeoff between the stores that affects CPU,
+memory, and storage characteristics. Compared to the Feather store, Parquet
+differs as follows:
+
+1. Parquet files occupy ~40% less space, which also reduces I/O pressure during
+   querying.
+2. Parquet utilizes more CPU cycles during ingest (~10%) and querying.
```

Comment:
Note that the Feather store behaves much better with queries that end early, because it allows for queries to be stopped early within a store, while our Parquet store only allows for stopping a query early after every store that has been evaluated. This is the primary reason why Parquet is not the default. Now while that is entirely fixable for Parquet, we simply haven't had the time to do so. We should caution about this, because for many parallel queries this makes a really large performance difference and improves latencies and responsiveness throughout the system.

Reply:
Added a note.